src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Subscales/Subscale/Subscale.tsx

```diff
@@ -1,12 +1,24 @@
 import { Fragment } from 'react';
+import { Box } from '@mui/material';
 
 import { Accordion } from 'modules/Dashboard/components';
+import { theme } from 'shared/styles';
 
 import { ReportTable } from '../../ReportTable';
-import { SubscalesTypes } from '../Subscales.const';
+import { AdditionalInformation } from '../AdditionalInformation';
+import {
+  AdditionalInformation as AdditionalInformationProps,
+  SubscalesTypes,
+} from '../Subscales.types';
 import { SubscaleProps } from './Subscale.types';
 
 export const Subscale = ({ items }: SubscaleProps) => {
+  const renderAdditionalInformation = (additionalInformation: AdditionalInformationProps) => (
```

Comment:
can you make this variable to a component? Using function instead of a component prevent react optimizations